src/foam/nanos/auth/AuthorizableAuthorizer.java

```diff
@@ -44,4 +44,28 @@ public void authorizeOnDelete(X x, FObject obj) throws AuthorizationException {
       ((Authorizable) obj).authorizeOnDelete(x);
     }
   }
+
+  public String createPermission(String op) {
+    return name + "." + op;
+  }
+
+  public boolean checkGlobalRead(X x) {
```

Comment:
See my comment below. I don't think `checkGlobalRead` and `checkGlobalRemove` are a good idea.